Drivers/Periphs/Src/radio.c

```diff
@@ -0,0 +1,61 @@
+#include "radio.h"
+
+#define CMD_SIZE 64
+
+static struct netif radio_Struct;
+static ip4_addr_t ipaddr;
+static ip4_addr_t netmask;
+static ip4_addr_t gw;
+osSemaphoreId radio_RXSemaphore;
+static struct link_str radio_arg;
+osThreadAttr_t attributes;
+static struct pbuf* radio_RXBuffer;
+static struct pbuf* radio_TXBuffer;
+
+void radio_Init(void){
```

Comment:
Not sure what the coding standards for this repo are, but typically we begin function names with caps. Like so
```c
void Radio_Init(void)
```